### PR TITLE
Tidy up duplicate and oddly named configuration properties

### PR DESCRIPTION
**Pull request by rich-nguyen**
An attempt to make some of the properties more readable, and update some of the legacy named keys.

Also relocated a few pieces of config which are admin-only.

This in turn allows us to use the new capi host.

### CODE REVIEW

**Comment by janua** [comment]
:+1:

**Comment by rich-nguyen** [comment]
Waiting for team city to have frontend.properties updated, WSA ticket raised

File: common/app/conf/switches.scala [hunk 1]
```diff
@@ -1243,20 +1243,16 @@ class SwitchBoardAgent(config: GuardianConfiguration) extends Plugin with Execut
...
+    services.S3.get(config.switches.key) map { response =>
+
+      val nextState = Properties(response)
+
+      for (switch <- Switches.all) {
+        nextState.get(switch.name) foreach {
+          case "on" => switch.switchOn()
+          case "off" => switch.switchOff()
+          case other => log.warn(s"Badly configured switch ${switch.name} -> $other")
+        }
```
Comment (johnduffell) [inline common/app/conf/switches.scala]: :+1: much better, no need to have 2 things pointing the same place

**Comment by johnduffell** [comment]
great, I guess people will need to update their local stuff on their dev boxes and also https://drive.google.com/drive/search?q=frontend.properties

**Comment by rich-nguyen** [comment]
yep, updated the doc, will send an email when it's out. thanks @johnduffell

**Pull request merged**